vraptor-core/src/main/java/br/com/caelum/vraptor/environment/DefaultEnvironment.java

```diff
@@ -101,7 +101,7 @@ private void loadAndPut(String environment) {
 			properties.load(resource);
 			LOG.debug("File {}.properties loaded", environment);
 		} catch (NullPointerException | IOException whenNotFound) {
-			LOG.warn("Could not find the file " + environment + ".properties to load.", whenNotFound);
+			LOG.debug("Could not find the file " + environment + ".properties to load.", whenNotFound);
```

Comment:
hey, @ferrelucas 
what do you think about keeping the warning, and move only the exception stack to the debug level?
something like this:
```
LOG.warn("Could not find the file " + environment + ".properties to load.");
LOG.debug("exception was " + whenNotFound);
```

Reply:
Yeah, that sounds better!